src/sql/connection.py

```diff
@@ -42,6 +43,10 @@
     "pymssql": "pymssql",
 }
 
+DIALECT_NAME_SQLALCHEMY_TO_SQLGLOT_MAPPING = {
```

Comment:
@edublancas 
The reason to have this mapping is the dialect name isn't fully consistent between sqlglot & sqlaclchemy, need this map to formalize the dialect name